app/src/main/res/values-nb-rNO/strings_settings.xml

```diff
@@ -3,10 +3,10 @@
 
     <!-- Favorites -->
     <string name="button_select_favorites">Velg favoritter</string>
-    <string name="button_sort_az">Sort A-Z</string>
+    <string name="button_sort_az">Sorter A-Å</string>
     <string name="hint_no_favorites_yet">Legg til favorittprogrammer her\nved å klikke på menyknappen.</string>
-    <string name="hint_modify_order_of_favorites">Endre rekkefølgen nedenfor (og rull ved bruk av sidene).</string>
-    <string name="hint_sort_favorites_az">The current order of favorites will be replaced by alphabetical order.</string>
+    <string name="hint_modify_order_of_favorites">Endre rekkefølgen nedenfor.</string>
+    <string name="hint_sort_favorites_az">Favorittene vil sorteres alfabetisk..</string>

```

Comment:
For the string "hint_sort_favorites_az" (which is displayed in a confirmation dialog), the idea I wanted to convey is that the current order will be replaced (in other words lost) by alphabetical order when validating, so that the user can click "Cancel" if it was a wrong click.
But maybe a shorter sentence is clear enough, I am not sure...

(Also, there is a double dot at the end of the sentence)

Reply:
In fact I think something such as "The current order of favorites will be replaced." should be enough for the warning.
Well, it's too late to update all translations for this release (will be published today), but I may change this in the next one.

Anyway, all three versions ("The current order of favorites will be replaced by alphabetical order.", "Favorites will be sorted alphabetically." and "The current order of favorites will be replaced.") are acceptable, even though the last one was probably the best for a warning.

Reply:
```suggestion
    <string name="hint_sort_favorites_az">Favorittene vil sorteres alfabetisk.</string>
```
Removing the double dot